.husky/commit-msg

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+head -1 $@ | grep -Eq "^\w{3}-\d+\s.+" || \
```

Comment:
@quesabe why is it fixed to 3? Should we do `{2,4}`? I saw an evidence of all of such cases.

Reply:
No problem. I just did not realise it might be something other than three letters.

Reply:
Ok, thanks.